### ISSUE TITLE
odysee-android-floss is not the FLOSS version of odysee-android

### ISSUE THREAD

**Issue by pixel2user** [OdyseeTeam/odysee-android-floss#9]
The naming scheme leads people to think that https://github.com/OdyseeTeam/odysee-android-floss is the FLOSS version of https://github.com/OdyseeTeam/odysee-android, which is not the case.

See #8

**Comment by BRBsoup** [comment]
> 

I think it's done, from newest readme file of odysee-android : 

Build Variants
There are two build variants: Full and FOSS. The Full variant uses Google APIs and the FOSS variant doesn't. You can switch between them on the "Build Variants" tab in Android Studio.

Whether or not  it's still the case of if that staddles the line of Build Variant or not or if they should just explicitly be named as two different apps, I'm not sure (not sure where that line would be either, to be honest... though I can see how that could be a subjective opinion as well)... though the versioning of both do seem to stay close, with FLOSS just slightly behind usually...

**Comment by BRBsoup** [comment]
> Sadly, it hasn't been done. In your own words: [#14 (comment)](https://github.com/OdyseeTeam/odysee-android-floss/issues/14#issuecomment-1355166414)

Yeah, it's actually all done and gone drove me about mad... sorry for presuming to correct you/second guessing you, I could have saved myself a dose of lost sanity if I just trusted your posts the first time I understood them!

Unfortunately this all is enough to seriously undermine the trust in an app that's drawn an image of itself as, and a community that that looks at it as, alternative platform to Big Bad Tech... However, the obvious is overlooked so egregiously that it might almost seem that the devs have just maybe perhaps purposefully cooked in some amount of discord into their own ecosystem. This isn't the first such misstep the devs took which has eroded trust, and I'm certainly not the first who has raised such points, therefore I do hope it's at least one of the lasts of this magnitude of gross negligence...
